pkgs/development/libraries/unixODBCDrivers/default.nix

```diff
@@ -47,7 +46,9 @@
 
     preConfigure = ''
       # we don't want to build a .pkg
-      sed -i 's/ADD_SUBDIRECTORY(osxinstall)//g' CMakeLists.txt
+      substituteInPlace CMakeLists.txt \
+        --replace "IF(APPLE)" "IF(0)" \
+        --replace "CMAKE_SYSTEM_NAME MATCHES AIX" "APPLE"
```

Comment:
I don't think we want this on Linux, do we?

Reply:
Indeed. Builds on Linux would be unaffected by manipulating macOS or AIX checks anyway.

Reply:
In cmake 0 == False, now I got it. I thought you had substituted the apple check for a tautology.